dotCMS/src/main/java/com/dotcms/rest/api/v1/relationships/RelationshipsResource.java

```diff
@@ -124,4 +132,60 @@ public final Response getOneSidedRelationships(
+     * @param request  - Http Request
+     * @param response - Http Response
+     * @param relationshipFieldVariable - Name of the relationship as defined in the structure.
+     * @param contentletIdentifier - Identifier of the contentlet
+     * @param luceneCondition - Extra conditions to add to the query. like +title:Some Title.  Can be
+     * @param limit 0 is the dotCMS max limit which is 10000. Be careful when searching for
+     * unlimited amount as all content will load into memory
+     * @param offset page to request
+     * @param orderby - Velocity variable name to sort by.  This is a string and can contain multiple
+     * values "sort1 acs, sort2 desc". Can be Null
+     * @return Returns empty List if no results are found
+     */
+    @GET
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    @Path("parents")
+    public Response pullRelatedParents(@Context final HttpServletRequest request,
+                                @Context final HttpServletResponse response,
+                                @QueryParam("fieldvariable")   final String relationshipFieldVariable,
+                                @QueryParam("identifier")   final String contentletIdentifier,
+                                @QueryParam(PaginationUtil.FILTER)                             final String luceneCondition,
+                                @DefaultValue("40") @QueryParam(PaginationUtil.PER_PAGE)       final int limit,
+                                @QueryParam(PaginationUtil.PAGE)                               final int offset,
+                                @DefaultValue("mod_date desc") @QueryParam(PaginationUtil.ORDER_BY) final String orderby) {
+
+            final InitDataObject initData = this.webResource.init(null, request, response, true, null);
+
+            Logger.debug(this, ()-> "Requesting pull related parents for the contentletIdentifier: " + contentletIdentifier +
+                                ", relationshipFieldVariable: " + relationshipFieldVariable + ", luceneCondition: " + luceneCondition +
+                                ", limit: " + limit + ", offset: " + offset + ", orderby: " + orderby);
+
+            final User user = initData.getUser();
+            final Language language = WebAPILocator.getLanguageWebAPI().getLanguage(request);
+            final long langId = UtilMethods.isSet(luceneCondition)
+                    && luceneCondition.contains("languageId") ? -1 : language.getId();
+            final String tmDate = request.getSession (false) != null?
+                    (String) request.getSession (false).getAttribute("tm_date"):null;
+            final PageMode mode = PageMode.get(request);
+            final boolean editOrPreviewMode = !mode.showLive;
+            final List<Contentlet> retrievedContentlets = ContentUtils
+                    .pullRelated(relationshipFieldVariable, contentletIdentifier,
```

Comment:
this method (or one of the other overloaded ones) can pull either children or parents.  Why should we limit this endpoint to only pulling parents?